src/Confluent.Kafka/WatermarkOffsets.cs

```diff
@@ -33,7 +33,7 @@ public class WatermarkOffsets
         ///     The offset of the earlist message in the topic/partition.
         /// </param>
         /// <param name="high">
-        ///     The offset of the last committed message in the topic/partition.
+        ///     The offset of the last stored message in the topic/partition.
         /// </param>
         public WatermarkOffsets(Offset low, Offset high)
```

Comment:
this is a general data structure used to store a low and high watermark offset values. it's returned by both the get and query methods.